app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -26,7 +40,20 @@ const Contacts = () => {
             placeholder: " Search Name, Email etc.",
           }}
         />
-        <Table />
+        <Table setshowDeleteAlert={setShowDeleteAlert} />
+        <NewContactPane
+          setShowPane={setShowNewContactPane}
+          showPane={showNewContactPane}
+        />
+        <Alert
+          closeButton={false}
+          isOpen={showDeleteAlert}
+          message="Are you sure you want to delete contact? These changes cannot be undone."
```

Comment:
Try and understand how this vastly improves the error message and point out to me what you have understood about the improvement.
```suggestion
message="Are you sure you want to delete this contact? This change cannot be undone."
```

Reply:
@yedhink 
It gives more clarity to end users. As using **pluralism** as in "changes" creates a doubt in mind of end user,
he/she may be confused whether what other than that particular contact is going to be vanished.

So, using **singular** as "change"  and "this" makes the motive of alert more clear on what's it about to do.

If there is anything else than please let me know so that i can improve in writing lines which interacts with end User.

Thanks.

Reply:
You are correct. In the suggested change we are being more `specific` rather than keeping things generic.